### PR TITLE
[Core] Disable runkratos

### PR DESCRIPTION
**Pull request by roigcarlo**
**📝 Description**
Disabling and deprecating runkratos. See #9733

**🆕 Changelog**
- Disabling runkratos

### CODE REVIEW

**Comment by EduardGomezEscandell** [comment]
You may also want to remove it from the README

File: kratos/python_scripts/run_tests.py [hunk 1]
```diff
@@ -177,7 +177,7 @@ def print_summary(exit_codes):
 
 def main():
     # Define the command
-    cmd = os.path.join(os.path.dirname(KtsUtls.GetKratosMultiphysicsPath()), 'runkratos')
+    cmd = sys.executable
```
Comment (philbucher) [inline kratos/python_scripts/run_tests.py]: Please use the fct from testing.utilities

Comment (roigcarlo) [reply kratos/python_scripts/run_tests.py]: mm I've been looking for it but There seems to be no function to do this, which one are you suggestion?

Comment (philbucher) [reply kratos/python_scripts/run_tests.py]: https://github.com/KratosMultiphysics/Kratos/blob/9ecfcecde03ff2c5fcf05040a1efadb505c8b8bc/kratos/python_scripts/testing/utilities.py#L9-L14

Comment (roigcarlo) [reply kratos/python_scripts/run_tests.py]: aaa thx!

File: CMakeLists.txt [hunk 2]
```diff
@@ -34,10 +34,7 @@ if(NOT DEFINED KRATOS_BINARY_DIR)
   set(KRATOS_BINARY_DIR ${CMAKE_BINARY_DIR})
 endif(NOT DEFINED KRATOS_BINARY_DIR)
 
-if(NOT DEFINED INSTALL_RUNKRATOS)
-  message("-- No INSTALL_RUNKRATOS is defined, setting to ON")
-  set(INSTALL_RUNKRATOS OFF)
-endif(NOT DEFINED INSTALL_RUNKRATOS)
+OPTION ( INSTALL_RUNKRATOS  "Installing deprecated runkratos" OFF )
```
Comment (roigcarlo) [inline CMakeLists.txt:37]: Oh, didn't know about this. 

We should change all the ifs we have with this !!

**Review by philbucher — DISMISSED** [review]

**Comment by loumalouomega** [comment]
Okay, I already did a branch with this..., I will remove the branch

**Comment by loumalouomega** [comment]
> Okay, I already did a branch with this..., I will remove the branch

It was this one... https://github.com/KratosMultiphysics/Kratos/tree/core/remove-runkratos

**Comment by roigcarlo** [comment]
It does not remove runkratos, only hides it. Also seems that I deleted more than intended. I will fix asap

**Comment by roigcarlo** [comment]
No longer relevant. See #11513

**Pull request closed without merging**